README.md

```diff
@@ -1,5 +1,21 @@
 ![](docs/images/logo.png)
 
+<p align="center">
+  <a href="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml">
+    <img alt="GitHub Workflow Status" src="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml/badge.svg?branch=main">
+  </a>
+  <a href="https://app.slack.com/client/T08PSQ7BQ/C052JRURD8V">
```

Comment:
Please update the latest link from README.md.

Reply:
Now updated with the latest slack link.